### ISSUE TITLE
I've discovered an AS which serves Fake Pantip Domains (FPDs)

### ISSUE THREAD

**Issue by kornkaobat** [easylist-thailand/easylist-thailand#74]
My investigation starts off when I clicked on a fake pantip post supposedly mimicking an intent to book bzx4 car. The behavior after initial compromise is alike to #54 but now the perpetrators limits 1 entry per IP. I immediately closed the website and ran an antivirus scan of the system compromised. Nothing was found but I might've been a victim of Zero-Days, who knows? Anyways, after pondering for some time, I've managed to find a way to scout for these malicious domains using advanced Google Search. 
<img width="554" alt="1" src="https://user-images.githubusercontent.com/26484978/201538764-c7fd367f-0737-4c40-a2a7-01475160657b.png">
Now what IPs are these sites pointing to?
<img width="370" alt="2" src="https://user-images.githubusercontent.com/26484978/201538853-fec97079-b2f6-465a-afaf-ae492bff6f4b.png">
<img width="337" alt="3" src="https://user-images.githubusercontent.com/26484978/201538855-2cc3a7eb-2e6f-49e0-80e9-c8e3aa1964b3.png">
Seems similar, let's investigate their AS.
<img width="622" alt="4" src="https://user-images.githubusercontent.com/26484978/201538892-03edd498-baba-4086-b169-a1a68e539273.png">
<img width="622" alt="5" src="https://user-images.githubusercontent.com/26484978/201538896-b70d352d-9b3e-459e-b9cb-5aa20b82ef93.png">
<img width="703" alt="6" src="https://user-images.githubusercontent.com/26484978/201538899-06cf0dc1-46d2-4fbd-9f6c-dc545c8d36d4.png">
<img width="634" alt="7" src="https://user-images.githubusercontent.com/26484978/201538900-b05bfc4b-469c-4464-bcb0-ee43bd094919.png">
AS 202984 ( Chernyshov Aleksandr Aleksandrovich )?
<img width="923" alt="8" src="https://user-images.githubusercontent.com/26484978/201538952-f8914f16-837c-42b8-8dc6-564276daef07.png">
13,056 IPs? I'm going to create a list for it.
<img width="265" alt="9" src="https://user-images.githubusercontent.com/26484978/201538999-239ec957-ef6d-413f-b3ef-c88b54e0fbda.png">
`46.8.141.0/24
193.105.4.0/24
46.8.29.0/24
109.172.112.0/22
95.182.120.0/22
92.63.203.0/24
185.231.246.0/23
45.139.236.0/24
185.176.25.0/24
45.139.237.0/24
2a0b:6cc0::/29
185.231.245.0/24
45.139.238.0/24
194.147.114.0/24
109.107.164.0/23
185.211.246.0/24
193.232.179.0/24
46.8.21.0/24
185.211.245.0/24
185.211.247.0/24
109.248.59.0/24
109.172.4.0/22
2a0b:6cc1:7::/48
185.211.170.0/24
195.19.92.0/24
92.63.199.0/24
185.231.244.0/24
45.139.239.0/24
194.38.10.0/24
195.133.60.0/22
109.172.108.0/22
62.76.142.0/24
31.184.240.0/23
185.211.244.0/24
46.8.158.0/24`
This is how many domains that will be blocked.
<img width="221" alt="10" src="https://user-images.githubusercontent.com/26484978/201539110-4aa55ca5-b1ea-4f17-9c9a-d47897296146.png">
In conclusion, can @kowith337 check if these IP ranges can be implemented in easylist-thailand? And, whether all of these IPs serves Fake Pantip Domains (FPDs) only.

**Comment by kowith337** [comment]
Unfortunately, I don't have any tools to deep check with IPs to get more domains, the method of me is using Image search, not web search, even I can turn on the 100 results per page.
My keyword term is adapted from you by these word...
`"หน้าของ สมาชิกหมายเลข" -site:pantip.com`

Then, I'll combo with either ViolentMonkey or Firemonkey (FF Only) and [GHHBD](https://www.jeffersonscher.com/gm/google-hit-hider/index.php) UserScript to check if it have anything new...

Note that searching on Google is love-hate for me due to uniqueness tracking parameters (e.g. `ei`, `ved`), sometime it's block me to use custom searching rules by show `sorry` page and throw Web Worker dialog to solve captcha! so I'm still rarely to use Google search nowadays.
